pyogrio/_ogr.pxd

```diff
@@ -214,7 +214,7 @@ cdef extern from "ogr_api.h":
     int             OGR_F_GetFieldAsDateTimeEx(OGRFeatureH feature, int n, int *y, int *m, int *d, int *h, int *m, float *s, int *z)
     double          OGR_F_GetFieldAsDouble(OGRFeatureH feature, int n)
     int             OGR_F_GetFieldAsInteger(OGRFeatureH feature, int n)
-    long            OGR_F_GetFieldAsInteger64(OGRFeatureH feature, int n)
+    int64_t         OGR_F_GetFieldAsInteger64(OGRFeatureH feature, int n)
```

Comment:
Good catch, looks like this should have originally been a `long long`, which would then have matched [Fiona](https://github.com/Toblerity/Fiona/blob/master/fiona/ogrext3.pxd#L338)

`int64_t` seems preferable because it is more explicit / less prone to error.

Would you mind updating `OGR_F_SetFieldInteger64` to `OGR_F_SetFieldInteger64(OGRFeatureH feature, int n, int64_t value)` to match, for symmetry?

Reply:
Sure!